cmake/tpp-mlir.cmake

```diff
@@ -32,6 +32,14 @@ if (TPP_MLIR_DIR)
     function(add_tpp_mlir_libs target)
         target_link_directories(${target} PRIVATE ${TPP_MLIR_DIR}/lib)
         target_link_libraries(${target} PRIVATE ${TPP_MLIR_LIBS})
+        target_link_options(${target} PRIVATE
+            -Wl,--no-as-needed
+            -L${TPP_MLIR_DIR}/lib
+            -ltpp_xsmm_runner_utils
+            -Wl,--as-needed
+        )
+        #FIXME: Provide platform-independent way of doing that:
+        install(FILES ${TPP_MLIR_DIR}/lib/libtpp_xsmm_runner_utils.so ${TPP_MLIR_DIR}/lib/libtpp_xsmm_runner_utils.so.19.0git DESTINATION ${OV_CPACK_RUNTIMEDIR})
```

Comment:
@rengolin, please suggest a proper alternative.

Reply:
That's actually not a bad idea, tbh. An alternative is to change the `setupvars.sh` to add the TPP build directory to the `LD_LIBRARY_PATH`.

Unless TPP can be installed as a proper library (on system path), there's not much else we can do.

Reply:
The idea is to have self-containing openvino package as it is now to model a final product without any extra dependencies. This is how binary size of the package will be calculated. This is one of the important product-level metrics.